<?xml version="1.0"?>
<launch>

	<param name="/stefmap_client/start_predictions" value="false"/>

	<node pkg = "tf" type = "static_transform_publisher" name="map_to_stefmap" args="0 0 0 0 0 0 /map /stefmap_frame 100" />

	<node pkg= "fremenarray" type = "fremenarray" name="fremenarray" />

	<node pkg= "stefmap_ros" type = "stefmap_server.py" name="stefmap_server" />

	<node pkg= "stefmap_ros" type = "stefmap_client_orkla_shortdata.py" name="stefmap_client" />

	<node pkg="rviz" type="rviz" name="rviz" />

	<node pkg="map_server" type="map_server" name="mapserver" args="$(find stefmap_ros)/maps/orkla-MS3.yaml"/>

	<node pkg="rqt_reconfigure" type="rqt_reconfigure" name="rqt_reconfigure"/>


</launch>